### DISCUSSION TITLE
======== Exception caught by widgets library ======================================================= The following UnimplementedError was thrown building WebViewView(dirty): createPlatformInAppWebViewWidget is not implemented on the current platform.

### DISCUSSION THREAD

**Discussion by muhsinzyne** [pichillilorenzo/flutter_inappwebview#2439] [Q&A]
======== Exception caught by widgets library =======================================================
The following UnimplementedError was thrown building WebViewView(dirty):
createPlatformInAppWebViewWidget is not implemented on the current platform.

The relevant error-causing widget was: 
  WebViewView WebViewView:file:///Users/muhsinzyne/work/flutter/lib/app/routes/app_pages.dart:100:19
When the exception was thrown, this was the stack: 
#0      InAppWebViewPlatform.createPlatformInAppWebViewWidget (package:flutter_inappwebview_platform_interface/src/inappwebview_platform.dart:90:5)
#1      new PlatformInAppWebViewWidget (package:flutter_inappwebview_platform_interface/src/in_app_webview/platform_inappwebview_widget.dart:209:40)
#2      new InAppWebView.fromPlatformCreationParams (package:flutter_inappwebview/src/in_app_webview/in_app_webview.dart:28:33)
#3      new InAppWebView (package:flutter_inappwebview/src/in_app_webview/in_app_webview.dart:295:13)
#4      WebViewView.build (package:ats_pro/app/modules/web-view/views/web_view_view.dart:16:13)
#5      StatelessElement.build (package:flutter/src/widgets/framework.dart:5557:49)
#6      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5487:15)
#7      Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#8      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469:5)
#9      ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463:5)
...     Normal element mounting (24 frames)
#33     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340:16)
#34     MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6904:36)
#35     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6916:32)
...     Normal element mounting (186 frames)
#221    Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340:16)
#222    MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6904:36)
#223    Element.updateChild (package:flutter/src/widgets/framework.dart:3849:18)
#224    Element.updateChildren (package:flutter/src/widgets/framework.dart:4036:32)
#225    MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6929:17)
#226    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#227    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#228    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650:11)
#229    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#230    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673:5)
#231    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#232    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#233    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#234    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816:5)
#235    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#236    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#237    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#238    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816:5)
#239    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:105:11)
#240    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#241    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#242    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650:11)
#243    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#244    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673:5)
#245    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#246    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#247    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#248    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816:5)
#249    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:105:11)
#250    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#251    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#252    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650:11)
#253    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#254    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673:5)
#255    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#256    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#257    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650:11)
#258    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#259    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673:5)
#260    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#261    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776:14)
#262    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#263    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776:14)
#264    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#265    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#266    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#267    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816:5)
#268    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#269    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#270    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#271    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816:5)
#272    Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)
#273    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)
#274    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650:11)
#275    Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)
#276    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2905:19)
#277    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:1136:21)
#278    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:443:5)
#279    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1392:15)
#280    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1313:9)
#281    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1171:5)
#282    _invoke (dart:ui/hooks.dart:312:13)
#283    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:419:5)
#284    _drawFrame (dart:ui/hooks.dart:283:31)
====================================================================================================

**Comment by pichillilorenzo** [comment]
Please add more info such as: plugin version, operating system/platform, flutter version, etc…
